src/dlgConnectionProfiles.cpp

```diff
@@ -716,166 +716,311 @@ QPair<bool, QString> dlgConnectionProfiles::writeProfileData(const QString& prof
+                   "non-PK, hack-and-slash game, cooperative rather than competitive in nature, and "
+                   "with a strong sense of community.");
+    }
+    if (hostUrl == QLatin1String("mudlet.org")) {
+        return qsl("This isn't a game profile, but a special one for testing Mudlet itself using "
+                   "Busted. You can also use it as a starting point to create automated tests for your "
+                   "own profiles!");
+    }
+    if (hostUrl == QLatin1String("luminarimud.com")) {
+        return qsl("Luminari is a deep, engaging game set in the world of the Luminari - A place where "
+                   "magic is entwined with the fabric of reality and the forces of evil and "
+                   "destruction are rising from a long slumber to again wreak havoc on the realm.  "
+                   "The gameplay of Luminari will be familiar to anyone who has played Dungeons and "
+                   "Dragons, Pathfinder or any of the many RPG systems based on the d20 ruleset.");
+    }
+    if (hostUrl == QLatin1String("stickmud.com")) {
+        return qsl("StickMUD is a free, medieval fantasy game with a graphical user interface and a "
+                   "depth of features. You are welcomed into the game world with maps and dashboards "
+                   "to complement your imagination. Newbies escape quickly into game play with minimal "
+                   "study time. Awaken under the wondrous Mallorn Tree in the center of Newbie Park "
+                   "and learn by playing. Challenge non-player characters to gain experience, advance "
+                   "level and maximize your stats. Between battles, sit on the enchanted bench under "
+                   "the Tree to rapidly heal and reduce wait time. Signs in the park present game "
+                   "features such as races, clans and guilds. Read up on teasers about the adventures "
+                   "on the path ahead like dragons, castles and sailing. Join a guild and learn the "
+                   "ways of a Bard, Fighter, Mage, Necromancer, Ninja, Thief, Healer or Priest. Train "
+                   "skills in both craft and combat aligned with your guild. Participate in frequent "
+                   "game-wide events to earn points exchanged for gold, experience or skill training. "
+                   "Heroes and villains alike are invited! Role play is optional and player vs. player "
+                   "combat is allowed in much of the game. StickMUD was born in Finland in June 1991 "
+                   "and is now hosted in Canada. Our diverse community of players and active game "
+                   "engineers are ready to welcome new players like you to one of the best text-based "
+                   "multi-player games ever!");
+    }
+    if (hostUrl == QLatin1String("reinosdeleyenda.es")) {
+// English translation (was used prior to switch to native Spanish of the MUD)
```

Comment:
"was used prior" etc. Could be dropped, unnecessary as part of git history

Reply:
You want just this line or the whole English translation removed?

Reply:
> Would love to see a future PR to toggle display language here as well.

There is no point IMVHO - someone is not going to want to read about a MUD that is both described AND played in a language that they cannot comprehend - if anything such a text will guide them away from such predefined MUDs...

Reply:
Also, the first line comment does explain why the remaining lines - though being valid code/text - IS commented out - it is one way to retain the English version of the text... 🤷‍♂️

Reply:
> You want just this line or the whole English translation removed?

The text in brackets only.